queries.sql

```diff
@@ -46,4 +46,22 @@ SELECT COUNT (*) FROM animals WHERE escape_attempts = 0;
 SELECT AVG(weight_kg) FROM animals;
 SELECT name, escape_attempts FROM animals WHERE escape_attempts = ( SELECT MAX(escape_attempts) FROM animals);
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+/* Part 3 */
+/* Animals belong to Melody Pond */
+SELECT animals.name FROM animals INNER JOIN owners ON (animals.owner_id = owners.full_name) WHERE owners.full_name = 'Melody Pond';
+/* List animals that are pokemon (their type is Pokemon). */
+SELECT animals.name FROM animals INNER JOIN species ON (animals.species_id = species.name) WHERE species.name = 'Pokemon';
+/* List owners & animals even those who don't have animals*/
+SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON (owners.full_name = animals.owner_id);
+/* Animals / species */
+SELECT species.name, COUNT(*) FROM species INNER JOIN animals ON (animals.species_id = species.name) GROUP BY species.name;
+/* Jennifer Orwell Digimonds */
+SELECT animals.name, species.name, owners.full_name FROM animals INNER JOIN species ON (animals.species_id = species.name) INNER JOIN owners ON (animals.owner_id = owners.full_name) WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+/* Dean's animal without escape_attempts */
+SELECT animals.name, animals.escape_attempts, owners.full_name FROM animals INNER JOIN owners ON (animals.owner_id = owners.full_name) WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
+/* Who owns the most animals? */
+SELECT owners.full_name, count (*) FROM animals INNER JOIN owners ON (animals.owner_id = owners.full_name) GROUP BY owners.full_name ORDER BY COUNT (*) DESC LIMIT 1;
```

Comment:
- [ ] Kindly use the Primary keys to join the tables. Even though your screenshots show the correct results, your queries are wrong since according to your schema, you are trying to match the ID with the names which can not be the same. Please match the IDs to the IDs. Like this `JOIN table1 ON (table2.id = table1.id)`, and not `JOIN table1 ON (table2.id = table1.name)`.
- [ ] When you correct your queries, please make sure they are running correctly and add the screenshots to the description. Your screenshots should show the query and the result.